README.md

```diff
@@ -20,9 +20,11 @@ To get started, see the following resources:
 * [Amazon Chime](https://aws.amazon.com/chime)
 * [Amazon Chime SDK Security](https://aws.amazon.com/blogs/business-productivity/understanding-security-in-the-amazon-chime-application-and-sdk/)
 * [Amazon Chime SDK Pricing](https://aws.amazon.com/chime/pricing/#Chime_SDK_)
+* [Amazon Chime SDK for JavaScript Supported browser](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers)
```

Comment:
`Amazon Chime SDK for JavaScript Supported Browsers`

Reply:
Updated